### ISSUE TITLE
Block release pull requests on verification run

### ISSUE THREAD

**Issue by KevinGrandon** [fusionjs/probot-app-workflow#72]
Currently we just mark it as a successful run once we trigger the verification pipeline. Instead let's make it pending until Buildkite is finished. This will result in increased stability in master.

**Comment by KevinGrandon** [comment]
This should be working now, but we are unable to deploy the latest probot which has these changes, will file a new issue for that.

**Comment by KevinGrandon** [comment]
Screenshot of waiting on verification:
![image](https://user-images.githubusercontent.com/122602/38518671-6b2e3cbc-3bf2-11e8-8390-58f41bd22180.png)
